### ISSUE TITLE
[BUG] Tank item model broken with Sodium and Indium

### ISSUE THREAD

**Issue by EcoBuilder13** [GabrielOlvH/Industrial-Revolution#153]
**Description:**
- The tank item model is broken with Sodium plus Indium. It works in the world so I do not know if there is a missing feature of the FRAPI that Indium has not added support for yet, a issue with the Sodium, or just a mistake in the model rendering code which breaks with Sodium.

**Steps to Reproduce:**
- Install [Sodium](https://github.com/CaffeineMC/sodium-fabric) from the `1.16.x/next` branch and [Indium](https://github.com/comp500/Indium).
- Get the tank item

**Technical Information:**
 - Minecraft: 1.16.5
 - Fabric API: 0.32.5
 - Industrial Revolution: 1.9.0-BETA
 - Sodium: `e0142381e9bbac0957c17679cb94eca035dd134e`

**Details:**
![image](https://user-images.githubusercontent.com/68478692/114323695-60b36780-9af4-11eb-85fe-e714376b81f8.png)

**Comment by Minecatr** [comment]
break it in survival

**Comment by GabrielOlvH** [comment]
is this still an issue? no clue why it happens, I can't see why it happens, might be best to report to the Indium dev.

**Comment by EcoBuilder13** [comment]
@GabrielOlvH, it didn't happen with 1.9.4 but it happened the version before. It is strange because without changing Sodium or Indium builds it occurs on certain versions but not others. Back in the 1.8.x versions of Indrev I had it happen once on a day I tested the mod and reported a couple of bugs to you but before I had the chance to report it you had pushed a new version since I was taking time to test it. In the new build it worked, then it broke again in 1.9.0. I think it might be an issue with Indium or the FRAPI. A while ago I had Secret Rooms and Phonos installed together and they worked but randomly while updating other mods I encountered https://github.com/FoundationGames/Phonos/issues/10. I did not change the versions and was using the same builds as in that issue report. I also tried downgrading the mods I just updated and it was still broken. It could be a problem with the models being built and cached but breaking for some reason. I would recommend contacting comp500 to see if he knows what might be happening.
